app/javascript/controllers/index.js

```diff
@@ -7,3 +7,43 @@ import { definitionsFromContext } from "stimulus/webpack-helpers"
 const application = Application.start()
 const context = require.context("controllers", true, /_controller\.js$/)
 application.load(definitionsFromContext(context))
+
```

Comment:
It would be good if we can move this code to a separate .js file. Perhaps we can get help from a TA?

Reply:
I cannot see the code you're refering to. Also, I pulled the latest version and the video is not showing..any idea why?